nlptest/nlptest.py

```diff
@@ -58,6 +58,9 @@ def __init__(
         super().__init__()
         self.task = task
 
+        if hub not in self.SUPPORTED_HUBS:
+            raise ValueError("Provided hub parameter is not in supported hubs. Please see docs for supported hubs.")
+
```

Comment:
It may be helpful to let users know supported hubs as follows:

```
 raise ValueError(f"Provided hub parameter is not in supported. Please choose one of supported hubs: {', '.join(self. self.SUPPORTED_HUBS)}")
```